src/wp-content/themes/twentytwentyfour/patterns/page-home-business-static.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * Title: Business home with static page
+ * Slug: twentytwentyfour/page-home-business-static
+ * Categories: twentytwentyfour_page
+ * Keywords: starter
+ * Block Types: core/post-content
+ * Post Types: page, wp_template
+ * Viewport width: 1400
+ * Description: A business home page with a hero section, a text section, a services section, a team section, a clients section, a FAQ section, and a CTA section.
+ */
+?>
```

Comment:
If this is intended to display a static page, why is there other content content than the static page?
This is part of the problem.
The user and as here, the developer can mix any type of content in any of these templates.